staging/src/k8s.io/apiextensions-apiserver/test/integration/validation_test.go

```diff
@@ -235,9 +235,9 @@ func TestCustomResourceValidation(t *testing.T) {
 			t.Fatal(err)
 		}
 
-		te, ns := "not-the-default-te", "not-the-default-ns"
+		ns := "not-the-default-ns"
 		for _, v := range noxuDefinition.Spec.Versions {
-			noxuResourceClient := newNamespacedCustomResourceVersionedClient(te, ns, dynamicClient, noxuDefinition, v.Name)
+			noxuResourceClient := newNamespacedCustomResourceVersionedClient(ns, dynamicClient, noxuDefinition, v.Name)
```

Comment:
Looks like you are reverting Chenqian's changes

Reply:
I have not picked out her change

Reply:
Please check https://github.com/futurewei-cloud/arktos/blob/master/staging/src/k8s.io/apiextensions-apiserver/test/integration/helpers.go.

I have picked up her latest change but it looks like not all the changes have been merged.